cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetInstanceService.java

```diff
@@ -399,4 +417,8 @@ private void ensureNamespaceExists(Id.Namespace namespace) throws Exception {
       }
     }
   }
+
+  private void publishAudit(Id.DatasetInstance datasetInstance, AuditType auditType) {
+    AuditPublishers.publishAudit(auditPublisher, datasetInstance, auditType, AuditPayload.EMPTY_PAYLOAD);
```

Comment:
JIRA for adding properties to audit payload - https://issues.cask.co/browse/CDAP-5220